src/frontend/js/components/PaymentButton/test-utils.ts

```diff
@@ -0,0 +1,5 @@
+export const formatPrice = (price: number, currency: string) =>
+  new Intl.NumberFormat('en', {
+    currency,
+    style: 'currency',
+  }).format(price);
```

Comment:
It seems useless.

Reply:
It's used in test, i've juste moved it there cause it's a better place.
I can replace it's usage by `Intl.NumberFormat`.

Reply:
fixed in [this commit](https://github.com/openfun/richie/pull/2105/commits/ab992a02d998d53ee80de933ac0df1dabbfce93a)